src/main/java/io/cdap/plugin/gcp/gcs/ServiceAccountAccessTokenProvider.java

```diff
@@ -56,7 +56,13 @@ public void refresh() throws IOException {
 
   private GoogleCredentials getCredentials() throws IOException {
     if (credentials == null) {
-      credentials = GCPUtils.loadCredentialsFromConf(conf);
+      if (conf != null) {
+        credentials = GCPUtils.loadCredentialsFromConf(conf);
+      } else {
+        // when initializing {@link ForwardingBigQueryFileOutputCommitter}
+        // conf is null.
```

Comment:
Please add more details in comments as in why is this necessary .

Reply:
Done.